src/zimg/colorspace/gamma.cpp

```diff
@@ -134,6 +137,26 @@ float srgb_inverse_eotf(float x) noexcept
 	return x;
 }
 
+float xvycc_eotf(float x) noexcept
+{
```

Comment:
xvYCC should be implemented as a mirrored Rec.709 OETF. In other words, copysign(rec709_oetf(fabs(x)), x).

Reply:
It is also still unclear if the standard gamut portion of xvYCC should be handled with Rec.1886, which is what would occur if the signal were to reach a non-xv TV.

Reply:
I tried using rec 709 but the colors when not display correctly with quicktime. I had to use the pure power bt1886 instead.

Regarding the last point we could maybe introduce a new range type? This is what the specs say on that regard

```Otherwise (TransferCharacteristics is equal to 11 (IEC 61966-2-4) or 12 (Rec. ITU-R BT.1361 extended colour gamut system)), E′R, E′G and E′B are analogue with a larger range not specified in this part of ISO/IEC 23001.```

or maybe let the applications deal with it?

Reply:
I converted a video containing xvycc with zimg and made sure colors matches the ones of the original video, can post pics if needed.

Reply:
>1.2^2.2 (BT.709) is less saturated than 1.2^2.4 (BT.1886).

The first one is almost sRGB. You are using almost sRGB (without linear segment) EOTF. It is what chrome does and many others, bit of course it is technically wrong. Sigh.

>needs to be compared to a reference xvYCC implementation.

The reference implementation uses to XYZ conversion. I cannot say whether BT.1886 can be used with xvYCC, because xvYCC was done before BT.1886, but the core of xvYCC (0 to 1 in float) should be using BT.1886, yeah.